shell/platform/embedder/fixtures/main.dart

```diff
@@ -9,6 +9,7 @@ import 'dart:isolate';
 import 'dart:ffi';
 import 'dart:core';
 import 'dart:convert';
+import 'dart:io';
```

Comment:
small cleanup: sort imports.

note to self: we need to figure out how to run the analyzer/linter over these fixtures.

Reply:
Done.